client-app/src/examples/portfolio/PortfolioPanel.ts

```diff
@@ -1,18 +1,29 @@
 import {hframe} from '@xh/hoist/cmp/layout';
 import {creates, hoistCmp} from '@xh/hoist/core';
 import {panel} from '@xh/hoist/desktop/cmp/panel';
+import {toolbar} from '@xh/hoist/desktop/cmp/toolbar';
 import {detailPanel} from './detail/DetailPanel';
 import {gridPanel} from './GridPanel';
 import {mapPanel} from './MapPanel';
 import {PortfolioPanelModel} from './PortfolioPanelModel';
+import {persistenceManager} from '@xh/hoist/desktop/cmp/persistenceManager';
 
 export const portfolioPanel = hoistCmp.factory({
     model: creates(PortfolioPanelModel),
 
     render() {
         return panel({
+            tbar: tbar(),
             mask: 'onLoad',
```

Comment:
Probably want to mask while the `PersistenceManagerModel` is loading views for the first time. Otherwise, this will render unmasked with whatever the default view is (in code) and then jump to the persisted view once it loads